lemur/plugins/lemur_aws/sns.py

```diff
@@ -57,4 +58,25 @@ def format_message(certificate, notification_type):
         "owner": certificate["owner"],
         "details": create_certificate_url(certificate["name"])
     }
+    if notification_type == "expiration":
+        json_message["notification_interval_days"] = calculate_expiration_days(options)
     return json.dumps(json_message)
+
+
+def calculate_expiration_days(options):
+    unit = get_option(options, "unit")
+    interval = get_option(options, "interval")
+    if unit == "weeks":
+        return interval * 7
+
+    elif unit == "months":
+        return interval * 30
+
+    elif unit == "days":
+        return interval
+
+
+def get_option(options, option_name):
```

Comment:
you might have already considered this, but would it be convenient to employ this:

https://github.com/Netflix/lemur/blob/98668b09722695c5c4f353c122359383edb1c811/lemur/plugins/base/v1.py#L113